dropwizard-jdbi3-transactions-core/pom.xml

```diff
@@ -10,7 +10,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>dropwizard-jdbi3-transactions-core</artifactId>
-    <version>3.4</version>
+    <version>3.5</version>
```

Comment:
Should we open another ticket about "un" forking this repo🙄 
Do you know the history here and the _why_ ?

Reply:
I'm not really sure the history on this one. I know we forked dropwizard itself because it didn't handle dual read/write database urls, but not sure why this repo exists

Reply:
https://campspot.atlassian.net/browse/CO-2370